.lintstagedrc

```diff
@@ -1,5 +1,6 @@
 {
   "*.js": [
+    "git add .",
```

Comment:
Think about this scenario:

1. You work on something
2. You've staged parts of some files and you're ready to commit
3. You run the linter, and `git add .` fucks up your stage. Now you have to unstage everything and carefully stage the lines/chunks/files you had before.